data/metadata.json

```diff
@@ -494,6 +494,25 @@
     "hackdex1/corrupted": {
         "sound-file": "hackdex1/roster-crash.webm"
     },
+    "hackdex2/board": {
+        "sound-file": "hackdex2/leviathan_board.webm",
+        "loop": true,
+        "type": "bg"
+    },
+    "hackdex2/articles": {
+        "sound-file": "hackdex2/leviathan_articles.webm",
+        "loop": true,
+        "type": "bg"
+    },
+    "hackdex2/hover": {
+        "sound-file": "hackdex2/leviathan_hover.webm"
```

Comment:
I guess this is the file provided by Design, but it seems a mistake since it sounds/looks more like a looping sound. I'll ask in Phabricator.

Reply:
Now I don't think this is a mistake, but this sound should be a looping sound.

Reply:
The bucle characteristics will be resolved in ticket https://phabricator.endlessm.com/T25322